plugins/trig/trig.go

```diff
@@ -0,0 +1,46 @@
+package trig
+
+import (
+	"math"
+
+	"github.com/influxdb/telegraf/plugins"
+)
+
+type Trig struct {
+	x         float64
+	Amplitude float64
+}
+
+var TrigConfig = `
+# Set the amplitude
+amplitude = 10.0
+`
+
+func (s *Trig) SampleConfig() string {
+	return TrigConfig
+}
+
+func (s *Trig) Description() string {
+	return "Insert trig data"
```

Comment:
This plugin will get compiled and included in any generated config file that users make, so I'm thinking that we might want to mention in the description that this is just a sample plugin for demonstration purposes.